lib/rubycritic/generators/html/base.rb

```diff
@@ -45,7 +45,7 @@ def root_directory
           @root_directory ||= Pathname.new(Config.root)
         end
 
-        def get_binding
+        def binding
           binding
```

Comment:
Won't this is lead to an infinite loop?

Reply:
Oops. Fixing.
I didn't perfectly understand the need for this function. Since it's private and can only be called in child classes, wouldn't it be just as effective to use `binding` directly in those classes instead of calling `get_binding`?

Reply:
Leaving this in for now, I don't think I have enough of an understanding to change this.

Reply:
Well, I'd say that we really don't need that method. I'm also not sure.
Give it a try - remove the method and see if the HTML reports are still properly generated.